unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/DecentralandEntity.cs

```diff
@@ -139,5 +129,17 @@ public void Cleanup()
             OnTransformChange = null;
             isReleased = true;
         }
+
+        public void UpdateInsideBoundariesStatus(bool isInsideBoundaries)
+        {
+            isInsideSceneBoundaries = isInsideBoundaries;
+            OnInsideBoundariesChanged?.Invoke(isInsideSceneBoundaries);
+        }
+
+        public void UpdateOuterBoundariesStatus(bool isInsideOuterBoundaries)
+        {
+            isInsideSceneOuterBoundaries = isInsideOuterBoundaries;
+            OnOuterBoundariesChanged?.Invoke(isInsideSceneOuterBoundaries);
+        }
```

Comment:
Do we need this? cannot we use `IOutOfSceneBoundariesHandler` like in `DCLAudioSource`?

Reply:
I tried using that but It wasn't working